netci.groovy

```diff
@@ -509,7 +509,7 @@ def innerLoopNonWindowsOSs = ['Ubuntu15.10', 'Debian8.2', 'OSX', 'CentOS7.1', 'O
             Utilities.addArchival(newBuildJob, "bin/build.pack,bin/osGroup.AnyCPU.${configurationGroup}/**,bin/ref/**,bin/packages/**,msbuild.log")
 
             // Use Server GC for Ubuntu/OSX Debug PR build & test
-            def useServerGC = ((os == 'Ubuntu15.10' || os == 'OSX') && configurationGroup == 'Release' && isPR)
+            def useServerGC = ((os == 'Ubuntu15.10') && configurationGroup == 'Release' && isPR)
```

Comment:
We can probably just remove this entirely from this definition. Originally we were using server GC on Ubuntu14.04 and OSX, and now both of those have moved to the other definition. I had switched it to Ubuntu15.10 as a replacement for Ubuntu14.04, so originally we weren't doing it on Ubuntu15.10 before m change.